Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1426,7 +1469,89 @@ -(void) textViewDidProgrammaticallyUpdate:(UITextView *)textView {
     [self resetAuxiliaryState];
 }
 
+- (void)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text isInsertion:(BOOL)isInsertion {
+    BOOL isCreatingOrStartingMention = self.state == HKWMentionsStartDetectionStateCreatingMention || [self isStringControlCharacter:text];
+    if (!isCreatingOrStartingMention) {
+        // Only implement Korean mentions fix when creating or starting mentions
+        // We will revisit a longer term alignment of using the fix in non-mentions cases, after we ramp the mentions fix
+        return;
+    }
+    self.suppressSelectionChangeNotifications = YES;
+    __strong __auto_type parentTextView = self.parentTextView;
+    unichar precedingChar = [parentTextView characterPrecedingLocation:(NSInteger)range.location];
+
+    if (self.nextInsertionShouldBeIgnored) {
+        self.nextInsertionShouldBeIgnored = NO;
+        self.previousSelectionRange = textView.selectedRange;
+        self.previousTextLength = [textView.text length];
```

Comment:
.length, also few places below

Reply:
not gonna change this because it's existing code and wanna limit what I add/delete to so i know what's wrong. created a ticket to clean up small things like this though